stand/defs.mk

```diff
@@ -38,7 +38,11 @@ WARNS?=		1
 BOOTSRC=	${SRCTOP}/stand
 EFISRC=		${BOOTSRC}/efi
 EFIINC=		${EFISRC}/include
+.if ${MACHINE_ARCH} == "amd64" && ${DO32:U0} == 1
```

Comment:
A quick comment here about WHY might be good.

Reply:
Because libefi uses it. Or do you mean a comment in the actual makefile?

Reply:
Would it be better to move the comment explaining DO32 up here?